RSSReader/View/FeedViewController.m

```diff
@@ -47,7 +47,10 @@ - (void)setupLayout {
 }
 
 - (void)loadNews {
-    [self.presenter loadNewsWithCompletion:^(NSError *error) {
+    UIApplication* app = [UIApplication sharedApplication];
```

Comment:
Looks like you might need this functionality in other controllers. Probably, it would be better to move this functionality into an extension to UIViewController. Please, create a func like -showActivitiIndicator:BOOL or two func for showing and hiding the indicator.
Use the parameter to decide wether to hide or show the activity indicator.

Reply:
fixed